mobile/src/bundles/employer/screens/candidates.tsx

```diff
@@ -1,9 +1,115 @@
+        englishLevel: 'Upper-Intermediate',
+        published: 'Published today',
+        hardSkills: [
+            'JavaScript',
+            'GitHub',
+            'NodeJS',
+            'React',
+            'Vite',
+            'React Native',
+        ],
+        badges: [
+            { label: BsaBadgeStepBadgesTitle.COMMUNICATION_SCORE, value: 7 },
+            { label: BsaBadgeStepBadgesTitle.PUNCTUALITY, value: 8 },
+            { label: BsaBadgeStepBadgesTitle.PROJECT_SCORE, value: 9 },
+        ],
+    },
+    {
+        id: 2,
+        salaryExpectation: 2500,
+        jobTitle: 'middle c++ developer',
+        location: 'Ukraine',
+        experienceYears: 1,
+        description:
+            'Responsible for driving revenue growth by identifying and pursuing new business opportunities, as well as maintaining relationships with existing clients text text text text text text text text text text',
+        englishLevel: 'Intermediate',
+        published: 'Published yesterday',
+        hardSkills: ['JavaScript', 'GitHub', 'NodeJS', 'React', 'React Native'],
+        badges: [
+            { label: BsaBadgeStepBadgesTitle.ENGLISH_LEVEL, value: 'B1' },
+            { label: BsaBadgeStepBadgesTitle.COMMUNICATION_SCORE, value: 7 },
+            { label: BsaBadgeStepBadgesTitle.LECTURE_SCORE, value: 8 },
+        ],
+    },
+    {
+        id: 3,
+        salaryExpectation: 2000,
+        jobTitle: 'middle javascript developer',
+        location: 'Ukraine',
+        experienceYears: 2.5,
+        description:
+            'Responsible for driving revenue growth by identifying and pursuing new business opportunities, as well as maintaining relationships with existing clients text text text text text text text text text text text',
+        englishLevel: 'Advanced',
+        published: 'Published 2 days ago',
+        hardSkills: ['JavaScript', 'GitHub', 'NodeJS', 'React', 'Vite'],
+        badges: [
+            { label: BsaBadgeStepBadgesTitle.PUNCTUALITY, value: 8 },
+            { label: BsaBadgeStepBadgesTitle.ENGLISH_LEVEL, value: 'C1' },
+            { label: BsaBadgeStepBadgesTitle.LECTURE_SCORE, value: 9 },
+        ],
+    },
+];
+const numberOfMockUsers = mockUsers.length;
 
 const Candidates: React.FC = () => {
-    return <Text>Employer screen: Candidates</Text>;
+    const [searchQuery, setSearchQuery] = useState('');
+
+    const filteredCandidates = useMemo(() => {
+        return mockUsers.filter(
+            (
+                user, //TODO replace with real users
+            ) =>
+                user.jobTitle.toLowerCase().includes(searchQuery.toLowerCase()),
+        );
+    }, [searchQuery]);
+
+    return (
+        <>
+            <StatusBar
+                barStyle="dark-content"
+                backgroundColor={Color.BACKGROUND}
+            />
+            <ScrollView
```

Comment:
It should be flat list, because there can be thousands of candidates 
Header should not be scrollable and search field also